Cargo.toml

```diff
@@ -82,8 +82,8 @@ async-compression = { version = "0.3.13", default-features = false, features = [
   "gzip",
   "tokio",
 ] }
-async-graphql = "7.0.7"
-async-graphql-axum = "7.0.7"
+async-graphql = "7.0.16"
+async-graphql-axum = "7.0.16"
```

Comment:
Upgraded these so that the patched version of `async-graphql` only differs from `7.0.16` by [this commit](https://github.com/yasamoka/async-graphql/commit/15fed95142019722e7da2c214ff9d8e2e755c9e5)